scripts/src/main/java/org/orbiswps/scripts/WpsScriptPlugin.java

```diff
@@ -145,45 +145,23 @@ public void unsetWpsClient(WpsClient wpsClient) {
      */
     @Activate
     public void activate(){
-        System.out.println("test");
         listIdProcess = new ArrayList<>();
         //Check the WpsService
         if(wpsServer != null){
+            //Mark the string to translate which will be used in the tree path.
+            I18N.tr("OrbisGIS");
```

Comment:
Use the method `I18n.marktr( )` in order to mark for translation without doing the translation.

@ebocher The translation of folder names are done on the fly in the tree folder. However the string have to be added to transifex, so at least one reference of the string have to be in the code.